nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/api/AbstractSession.java

```diff
@@ -148,6 +148,9 @@
 
     public static final String BINARY_TEXT_SYS_PROP = "fulltextBinary";
 
+    // @since 10.2
+    public static final String BINARY_TEXT_PREFIX = "TEXT_";
```

Comment:
Please use `/** @since 10.2 */` so that it's a full Javadoc. You may also mention a description like `Prefix added to the system property name in the event properties for the {@link DocumentEventTypes#BINARYTEXT_UPDATED} event. The value is the actual fulltext.`